shared/video-sdk/develop/use-an-extension/project-implementation/flutter.mdx

```diff
@@ -0,0 +1,41 @@
+<PlatformWrapper platform="flutter">
+
+This section presents the framework code you add to your <Vpl k="NAME" /> project to integrate an extension.
+
+1.  **Load the extension provider**
+
+    You call `loadExtensionProvider` during <Vg k="ENGINE" /> initialization to specify the extension library path. To do this, add the following code to the `setupVideoSDKEngine` method after you initialize the engine with `agoraEngine.initialize`:
+
+    ``` dart
+    agoraEngine.loadExtensionProvider("<extensionLibraryPath>");
+    ````
+
+1.  **Enable the extension**
+
+    To enable the extension, add the following code to the `setupVideoSDKEngine` method before `await agoraEngine.enableVideo();`:
+
+    ``` dart
+    agoraEngine.enableExtension(
+        provider: "<The name of the extension provider>",
+        extension: "<extensionName>",
+        type: MediaSourceType.unknownMediaSource,
+        enable: true
+    );
+    ```
+    If the extension uses the second camera to capture video, set the type parameter to `secondaryCameraSource`.
+
+    To enable multiple extensions, call `enableExtension` as many times. The sequence of enabling extensions determines the order of these extensions in the transmission pipeline. For example, if you enable extension A before extension B, extension A processes data from the <Vg k="VSDK" /> before extension B.
```

Comment:
call `enableExtension` for each extension. The calls to `enableExtension` determines the order of each extension in the transmission pipeline. For example, if you enable extension A before extension B, extension A processes data from the <Vg k="VSDK" /> before extension B.

Reply:
done